src/gobupload/relate/__init__.py

```diff
@@ -246,22 +245,18 @@ def build_relations(msg):
     else:
         logger.info(f"** Relate {catalog_name} {collection_name} {attribute_name}")
 
-        try:
-            relate_update(catalog_name, collection_name, attribute_name)
-            relation_name = get_relation_name(GOBModel(), catalog_name, collection_name, attribute_name)
-
-            msg["header"].update({
-                "catalogue": "rel",
-                "collection": relation_name,
-                "entity": relation_name,
-                "original_catalogue": catalog_name,
-                "original_collection": collection_name,
-                "original_attribute": attribute_name,
-            })
-
-            return msg
-        except Exception as e:
-            _log_exception(f"{attribute_name} update FAILED", e)
+        relation_name = get_relation_name(GOBModel(), catalog_name, collection_name, attribute_name)
```

Comment:
Misschien een uitleg erbij waarom we een lege stap in stand laten en de stap niet gewoon overslaan in de workflow? Of is de stap niet leeg en heeft deze nog een functie?

Reply:
Deze stap wordt ook gebruikt om de relate job te splitsen in de verschillende losse taken als dat nodig is.

Reply:
OK, dan stel ik voor om hier commentaar toe te voegen